### DISCUSSION TITLE
The format name `typst`?

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#9483] [Q&A] [answered]
### Description

Currently the format name for rendering PDF using the Typst CLI is `typst`, but is this ideal when compared to other formats?

In other words, the other formats are not intermediate formats, but the final output is the name of the format.
For example, `hugo-md` (the final output is hugo web site), `docusaurus-md` (the final output is docusaurus web site).

So shouldn't the format to output PDF with typst as intermediate format be `pdf-typst`?

Since the Typst roadmap includes compiling to other formats like html, I think `pdf-typst` makes more sense than `typst`, considering that there may be future demand for HTML documents using typst as an intermediate format.
Shouldn't the format name `typst` be used to generate typst files without rendering PDF?

**Comment by cscheid** [comment]
Yes, that's currently an inconsistency on our side. It turns out to be hard to solve because of other assumptions in our code base wrt `pdf` being a format "owned" by the pdflatex engine. So, we'll eventually have to fix it, but it's not going to happen soon.

**Reply by eitsupi** [reply]
Thank you for answering. That seems reasonable.

Perhaps this should be converted into an issue?

**Reply by cderv** [reply, marked as answer]
I would say this is tracked already at
* https://github.com/quarto-dev/quarto-cli/issues/7039

**Reply by eitsupi** [reply]
Glad to know that, thanks!
